config/crd/bases/nodeobservability.olm.openshift.io_machineconfigs.yaml

```diff
@@ -0,0 +1,70 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: machineconfigs.nodeobservability.olm.openshift.io
+spec:
+  group: nodeobservability.olm.openshift.io
+  names:
+    kind: Machineconfig
+    listKind: MachineconfigList
+    plural: machineconfigs
+    singular: machineconfig
+  scope: Namespaced
```

Comment:
I think those should be cluster resources, right?

Reply:
NodeObservabilityMachineConfig changed to cluster resource in [d0805fd](https://github.com/openshift/node-observability-operator/pull/16/commits/d0805fd9c9a88344d99aa95ea52cf86ac8ad105c)